clang/test/SemaCUDA/launch_bounds.cu

```diff
@@ -12,7 +12,7 @@ __launch_bounds__(0x10000000000000000) void TestWayTooBigArg(void); // expected-
 __launch_bounds__(-128, 7) void TestNegArg1(void); // expected-warning {{'launch_bounds' attribute parameter 0 is negative and will be ignored}}
 __launch_bounds__(128, -7) void TestNegArg2(void); // expected-warning {{'launch_bounds' attribute parameter 1 is negative and will be ignored}}
```

Comment:
We need a test for negative max blocks argument.

Reply:
Done in 028d270290218f3cc4fb35acc721b0645f2118ea